src/commands/build.rs

```diff
@@ -1,33 +1,32 @@
 use std::path::PathBuf;
 
+use anyhow::Result;
+use clap::Parser;
+
```

Comment:
Official request for bikeshedding (that should be ignored in this PR).

Is there a convention on order imports that we want to follow?